src/emc/usr_intf/axis/scripts/axis.py

```diff
@@ -3340,12 +3350,17 @@ def units(s, d=1.0):
 
 kinsmodule=inifile.find("KINS", "KINEMATICS").lower()
 kins_is_trivkins = False
+trivkinstype = "none"
 if kinsmodule.split()[0] == "trivkins":
     kins_is_trivkins = True
     trivkinscoords = "XYZABCUVW"
     for item in kinsmodule.split():
         if "coordinates=" in item:
             trivkinscoords = item.split("=")[1]
+        if "kinstype=" in item:
+            trivkinstype = item.split("=")[1]
+        else:
+            trivkinstype = "single"
```

Comment:
Being new at this git thing I may (probably) have confused the issue with my response.
I thought https://github.com/LinuxCNC/linuxcnc/pull/261/commits/681cf92b75096e989061ac29edcfe107370a8ad4 would address this problem.
Should I close this pr and present a new one with the complete changes?